api/src/main/java/jakarta/data/expression/NumericExpression.java

```diff
@@ -257,6 +258,46 @@ default <U extends T> NumericExpression<U, N> divide(
         return NumericOperatorExpression.of(DIVIDE, this, divisorExpression);
     }
 
+    // TODO Javadoc
+    default NumericExpression<T, N> plus(
+            NumericLiteral<Object, N> addendLiteral) {
+        @SuppressWarnings("unchecked")
+        NumericExpression<T, N> literal =
+            (NumericExpression<T, N>) addendLiteral;
+
+        return NumericOperatorExpression.of(PLUS, this, literal);
+    }
+
+    // TODO Javadoc
+    default NumericExpression<T, N> minus(
+            NumericLiteral<Object, N> subtrahendLiteral) {
```

Comment:
This is a layer breaker. We don't want to leak `NumericLiteral` onto the API of `NumericExpression`.